tools/scripts/check_test_ids.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# This script does a sanity check for the number of tests and the number of test IDs and
+# checks for unique test IDs
+
+## sanity check for number of tests and test IDs
```

Comment:
I would recommend to add a `set -ex` for exiting at failing commands and printing them.

Reply:
@dkroenke how about only `set -e`? With the `x` parameter it will always be printed in the hooks and there is already an output with the numbers if the check fails